front-end/src/util/__tests__/getEncodedAddress.test.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019-2020 @paritytech/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import getEncodedAddress from '../getEncodedAddress';
+
+jest.mock('../getNetwork', () => jest.fn(() => {return 'polkadot';}));
+const getNetwork = require('../getNetwork');
+
+describe('Get Encoded Address', () => {
+	beforeEach(() => {
+		jest.resetModules();
+	});
+
+	const address1 = 'FqfW96FuXRH7pRy9njeNSxu4CNoBM1hKnVAmNRrFrU2izwj';
+
+	it('for Kusama', () => {
```

Comment:
Same here, when we have many tests, and some are failing, we need to understand right away what is failing.